baremetal_benchmarking/workloads/tensorflow_workload.py

```diff
@@ -27,7 +27,7 @@ def download_workload(self, test_config_dict):
         print("\n-- Installing Tensorflow..")
         utils.exec_shell_cmd(TENSORFLOW_INSTALL_CMD)
 
-    def download_bert_models(self):
+    def download_bert_models(self, test_config_dict):
```

Comment:
Do we need this new function parameter test_config_dict?

Reply:
Done